### DISCUSSION TITLE
Angular v19 with no app.config.ts, how to install primeng v19

### DISCUSSION THREAD

**Discussion by reagustin** [primefaces/community#3278] [PrimeNG]
Hello, I was reading the getting started docs and found out that the information there does not adapt to the actual structure i got for my application. There is no app.config.ts and the routing is handled as the old style, i tried installing an older version like prime ng v15 lts or v16 lts but it does not allow me to do so. I don't want to force the install, is there any documentation on how to install in angular v19 with the old structure/old routing?

**Comment by PHuhn** [comment]
I upgraded from Angular 18 and PrimeNG 17, without using app.config.ts. I placed the theming/style in the app.component.ts. I have a limited [StackBlitz](https://stackblitz.com/edit/dtdh1e2x?file=src%2Fapp%2Fdialog-basic-demo.ts) example.
